draft-ietf-quic-recovery.md

```diff
@@ -784,7 +785,7 @@ If a server accepts 0-RTT, but does not buffer 0-RTT packets that arrive
 before Initial packets, early 0-RTT packets will be declared lost, but that
 is expected to be infrequent.
 
-It is expected that keys are discarded after packets encrypted with them would
+It is expected that keys are discarded after all packets encrypted with them would
```

Comment:
I'm not sure if this is correct. You could receive HANDSHAKE_DONE without having received all ACKs for Handshake packets.

Reply:
Hmm.  I considered s/all/any/ but am not sure that helps.  The best idea I have right now (just before bedtime...) is to add to the next sentence that Handshake keys are discarded when the handshake is confirmed.